fe/fe-core/src/test/java/com/starrocks/sql/analyzer/AnalyzeTestUtil.java

```diff
@@ -188,6 +188,19 @@ public static StarRocksAssert getStarRocksAssert() {
         return starRocksAssert;
     }
 
+    public static StatementBase analyzeOnceSuccess(String originStmt) {
```

Comment:
why not use analyzeSuccess?

Reply:
Because analyzeSuccess will test with view again, and It will expand "INTERVAL n type" with throw exception, so I change to use analyzeWithoutTestView instead.

Reply:
> why not use analyzeSuccess?

Because Before this PR We call visitTimestampArithmeticExpr in AST2SQL, after this PR we visit visitFunctionCall(**not TimestampArithmeticExpr**), so the form after expansion is different. So we should only test analyzeWithoutTestView.